build-config/src/main/resources/forbidden-runtime.txt

```diff
@@ -0,0 +1,36 @@
+# Hibernate Search, full-text search for your domain model
+#
+# License: GNU Lesser General Public License (LGPL), version 2.1 or later
+# See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+
+# This file contains forbidden API signatures common to all modules
+
+@ignoreUnresolvable
+@defaultMessage Should not be used
+
+################################################################################################################
+# Some packages that should not be used
+java.awt.**
+sun.**
+org.slf4j.**
+junit.framework.**
+
+################################################################################################################
+# org.testng.Assert#fail() should be used instead of:
```

Comment:
This one looks weird, considering you're using JUnit on Search :).

Reply:
it's a comment :)